src/expr.rs

```diff
@@ -1970,6 +1972,13 @@ fn choose_rhs<R: Rewrite>(
                     Some(format!("{}{}", new_indent_str, new_rhs))
                 }
                 (None, Some(ref new_rhs)) => Some(format!("{}{}", new_indent_str, new_rhs)),
+                (None, None) if rhs_tactics == RhsTactics::AllowOverflow => {
+                    let shape = Shape {
+                        width: 8096, // 8096 is close to inifinite for rustfmt.
```

Comment:
Can you extract this value to a `const` ?

```suggestion
                        width: 8096, // 8096 is close to infinite for rustfmt.
```

Reply:
Fixed it in https://github.com/rust-lang/rustfmt/pull/3566/commits/05c34a80ca26bb96c9f20b8e43e717911eddbb99 and https://github.com/rust-lang/rustfmt/pull/3566/commits/08f9ce40bfaf9d5e73a1907954a10fb2ac0d9ef8.